### PR TITLE
Catpuccin Mocha Theme

### PR DESCRIPTION
**Pull request by julve**
Added the catpuccin mocha theme for the archive.

### CODE REVIEW

File: Themes/Catpuccin mocha.theme.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+    "name": "Catpuccin_Mocha",
+    "style": "Dark",
+    "author": [
+        {
+            "name": "Catpuccin Mocha",
+            "url": "https://catppuccin.com/"
+        }
```
Comment (DivineDominion) [inline Themes/Catpuccin mocha.theme.json:8]: Referencing the original is a cool idea. Please feel free to also add yourself for credit where credit is due!

**Comment by DivineDominion** [comment]
Thanks for the submission! Could you paste screenshots to this PR in a comment? We're not displaying them in the app, yet, but consider doing so in the future, and a visual reference would be nice. 👍

**Comment by julve** [comment]
Sure! Are this enough?
CleanShot 2024-08-14 at 09 35 46@2x
CleanShot 2024-08-14 at 09 36 17@2x
CleanShot 2024-08-14 at 09 36 47@2x

File: Themes/Catpuccin mocha.theme.json (hunk 1 above)
Comment (julve) [reply Themes/Catpuccin mocha.theme.json:8]: Oh I don't know where should I add my nickname.

**Comment by DivineDominion** [comment]
@julve Sorry for the delay! Can you check with @rogeruiz's #19 and the repo at https://github.com/catppuccin/the-archive if you want to incorporate any changes? Their theme suite, while coming later to the game, is more extensive -- so I'd want to merge that instead and combine yours with it where possible.

**Comment by julve** [comment]
Hi @DivineDominion I have no problem incorporating the suite he's bringing.

**Comment by DivineDominion** [comment]
Thank you!

Replaced by #19

**Pull request closed without merging**